presto-hive/src/main/java/com/facebook/presto/hive/HiveClientModule.java

```diff
@@ -161,6 +168,9 @@ public void configure(Binder binder)
 
         binder.bind(CacheStats.class).in(Scopes.SINGLETON);
         newExporter(binder).export(CacheStats.class).withGeneratedName();
+
+        configBinder(binder).bindConfig(LocalRangeCacheConfig.class);
+        configBinder(binder).bindConfig(AlluxioCacheConfig.class);
```

Comment:
It's weird to have `AlluxioCacheConfig` binding here and `AlluxioCachingConfigurationProvider` binding in another module. Can we have a single `CacheModule` and put cache related binding there or just remove `AlluxioCachingModule`

Reply:
Yeah, lets remove AlluxioCachingModule, and Introduce a CachingModule in the presto-cache. Using this new module for all cache artifacts. cacheConfig, LocalRangeCacheConfig, AlluxioCacheConfig, AlluxioCachingConfigurationProvider

Reply:
Also CacheStats, etc.

Reply:
Moved all caching bindings into `CachingModule`. Not sure how to move `CachingModule` to `presto-cache` given `AlluxioCachingConfigurationProvider` depends on types in `presto-hive`. @jainxrohit Any idea?

Reply:
We can extract HdfsContext from the HdfsEnvironment, and move it along with DynamicConfigurationProvider to presto-hive-common. That should solve this issue?

Reply:
Thanks @jainxrohit! 
Moved `CachingModule` to `presto-cache` along with `AlluxioCachingConfigurationProvider`.